packages/shared/src/components/comments/MainComment.tsx

```diff
@@ -32,6 +35,7 @@ const shouldShowBannerOnComment = (
 
 export default function MainComment({
   className,
+  commentBoxClassName,
```

Comment:
Open for this, but generally we prefer the object method of classNames when more then 1 are needed.